experimental/css.html

```diff
@@ -0,0 +1,127 @@
+<style>
+
+  .branding-below {
+    bottom: 56px;
+    top: 0;
+  }
+  
+  .heading {
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0;
+    margin-inline-end: 0;
+  }
+  
+  .btn-add {        
+    display: inline;
+    float: right; 
+    height: 20px; 
+    line-height: 20px;
+    min-width: 40px; 
+    max-width: 40px; 
+  }
+  
+  .btn-delete1 {     
+    display: inline;
+    float: right;
+    height: 20px; 
+    line-height: 20px;
+    margin-top: 3px;
+    min-width: 52px; 
+    max-width: 52px; 
+    padding: 0;
+    position: absolute;
+    right: 18px;
+    text-align: center;
+  }
+  
+  .btn-delete2 {     
+    display: inline;
+    float: right;
+    height: 20px; 
+    line-height: 20px;
+    min-width: 52px; 
+    max-width: 52px; 
+    padding: 0;
+    position: absolute;
+    right: 18px;
+    text-align: center;
+  }
+  
+  .btn-del-filter {  
+    display: block;
+    float: right;
+    height: 20px; 
+    line-height: 20px;
+    margin-bottom: 10px;
+    min-width: 20px; 
+    max-width: 20px; 
+    padding: 0;
+    text-align: center;
+  }
+  
+  .width90 {
+    min-width: 90px;
+    max-width: 90px;
+  }
+
+  .width100 {
+    min-width: 100px;
+    max-width: 100px;
+  }
+  
+  .width125 {
+    min-width: 125px;
+    max-width: 125px;
+  }
+  
+  .width130 {
+    min-width: 130px;
+    max-width: 130px;
+  }
+  
+  .boxed {                       
+    background: white;
+    border: 2px solid #e5e5e5;
+    border-radius: 6px;
+    box-shadow: 2px 2px #D3D3D3;
+    padding: 5px;
+    padding-bottom: 20px;   
+  }
+   
+  .box-slice {
+    border: 1px solid #e5e5e5;
+    padding: 10px;
+  }
+  
+  .drp1 {
```

Comment:
nit: Add a description to the css class. how is it different from btn-delete2, or can modify the class name to be more obvious.

Reply:
Done